src/main/java/com/bandwidth/sqs/consumer/SqsConsumer.java

```diff
@@ -36,54 +34,54 @@
 
 import static com.bandwidth.sqs.consumer.acknowledger.MessageAcknowledger.AckMode;
 
-public class Consumer {
+public class SqsConsumer {
```

Comment:
I get that this class is a central element, but it is Yuge (to quote a famous US president) but not in a good way. I haven't thought about how to break this class apart but it scream: refactor me, make me smaller, give me fewer things to do.